biojava-structure/src/main/java/org/biojava/nbio/structure/io/mmcif/ZipChemCompProvider.java

```diff
@@ -293,7 +293,7 @@ private synchronized boolean addToZipFileSystem(Path zipFile, File[] files, Path
 		*/
 
 		// Copy in each file.
-		try (FileSystem zipfs = FileSystems.newFileSystem(zipFile, null)) {
+		try (FileSystem zipfs = FileSystems.newFileSystem(zipFile, (ClassLoader) null)) {
```

Comment:
This line fixes an error I was getting compiling with Java 1.8:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project biojava-structure: Compilation failure
[ERROR] /Users/bliven_s/git/biojava_ws/biojava/biojava-structure/src/main/java/org/biojava/nbio/structure/io/mmcif/ZipChemCompProvider.java:[296,52] reference to newFileSystem is ambiguous
[ERROR]   both method newFileSystem(java.nio.file.Path,java.lang.ClassLoader) in java.nio.file.FileSystems and method newFileSystem(java.nio.file.Path,java.util.Map<java.lang.String,?>) in java.nio.file.FileSystems match
```

The second form isn't documented, so it could be that this change will cause a warning with different JDKs.

Reply:
@sbliven 
The change to TreeMap still shows the test to be flaky. I did try this initially, but it doesn't fix the flaky error.

Reply:
@sbliven 
I did initially try to fix this error with the same change. However the TreeMap here, is not fixing this flaky error. The tool still marks it as flaky with error of the sort -
```
[ERROR] Failures: 
[ERROR]   AlignmentToolsTest.testToConciseAlignmentString:342 1. Cycles. expected:<[1>2>3>1] 7>7> but was:<[2>3>1>2] 7>7>
[INFO] 
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0
```

Reply:
@sbliven @josemduarte 
Would like just to link the Nondex tool here : https://github.com/TestingResearchIllinois/NonDex

The tool requires to add the plugin as specified in the documentation and run with maven for testing. Ideally I would run this as - 
` mvn -pl biojava-integrationtest nondex:nondex -Dtest=AlignmentToolsTest#testToConciseAlignmentString`

It could be used to run for the whole module as well -
`mvn -pl biojava-integrationtest nondex:nondex`.

This could be added into travis.yml, so it can be run on new tests too.

It also has the debug feature that can be used to point to the file(s) that could require changes. The link to the tool has all the documentation.